target/scripts/check-for-changes.sh

```diff
@@ -6,37 +6,34 @@
 # shellcheck source=./helpers/index.sh
 source /usr/local/bin/helpers/index.sh
 
-function _log_date
-{
-  date +"%Y-%m-%d %H:%M:%S"
-}
-
 function _changedetector_notify
 {
-  local LOG_LEVEL_ARGUMENT="${1}"
+  local LEVEL="${1}"
   shift 1
 
-  _notify "${LOG_LEVEL_ARGUMENT}" '[ CHANGEDETECTOR ]' " $(_log_date) " "${*}"
+  _notify "${LEVEL}" '[ CHANGEDETECTOR ]' " $(date +"%Y-%m-%d %H:%M:%S") " "${*}"
 }
 
-LOG_DATE=$(_log_date)
-_changedetector_notify 'debug' "${LOG_DATE} Start check-for-changes script."
+_changedetector_notify 'debug' 'Starting changedetector'
 
 # ? --------------------------------------------- Checks
 
-cd /tmp/docker-mailserver || exit 1
+if ! cd /tmp/docker-mailserver &>/dev/null
+then
+  _changedetector_notify 'error' "Could not change into '/tmp/docker-mailserver' directory"
+  exit 1
+fi
 
-# check postfix-accounts.cf exist else break
 if [[ ! -f postfix-accounts.cf ]]
 then
-  _changedetector_notify 'info' "${LOG_DATE} postfix-accounts.cf is missing! 'check-for-changes.sh' will exit."
+  _changedetector_notify 'info' "postfix-accounts.cf is missing! 'check-for-changes.sh' will exit."
   exit 0
 fi
 
-# verify checksum file exists; must be prepared by start-mailserver.sh
+# checksum file must have been prepared by start-mailserver.sh
 if [[ ! -f ${CHKSUM_FILE} ]]
 then
-  _changedetector_notify 'info' "${LOG_DATE} ${CHKSUM_FILE} is missing! Start script failed? Exit!"
+  _changedetector_notify 'info' "${CHKSUM_FILE} is missing! 'check-for-changes.sh' will exit."
```

Comment:
These are the only two logs in this script that use a `.` at the end. I know you love things being consistent, so just pointing that out :sweat_smile:

Reply:
I adjusted it :D

Reply:
> I adjusted it :D

In this PR? There's no new commits since this review.

Reply:
Done in cce33f80d545c23eaddb67a4afb2a2bd9596e0d9